src/hooks/useAnalytics.ts

```diff
@@ -54,6 +55,11 @@ export const useAnalytics = () => {
     identify(AnalyticsUserProperty.Locale, selectedLocale);
   }, [selectedLocale]);
 
+  // AnalyticsUserProperty.Version
+  useEffect(() => {
```

Comment:
might be worth validating `latestTag` is not undefined

Reply:
Also I think its probably ok to just set it as the whole tag instead of striping out the number.

Reply:
ah so I stripped out the number specifically so that it'd be in a format that amplitude can perform comparisons on (i.e. filtering to logs after v1.1.1). Based on my understanding from documentation [here](https://help.amplitude.com/hc/en-us/articles/360017800371-Releases-See-how-users-respond-to-changes-in-your-product#h_939808537151539301617278), I have to strip it! 

will def make the undefined change tho